params/query_dev.yml

```diff
@@ -3,14 +3,21 @@ md_keywords:
 - molecular dynamic
 - molecular-dynamics
 - molecular-dynamic
-- namd
 - md trajectory
 - md trajectories
 - md simulation
 - md simulations
-- amber
 - gromacs
+- gromos
+- namd
+- amber
```

Comment:
Do we really want to have all different amber options in the query_dev?

Reply:
Not at all. I will simplify.